cytoscape-panzoom.js

```diff
@@ -407,13 +451,52 @@ SOFTWARE.
             return false;
           });
 
+          sliding = false;
+          var sliderTdownHandler, sliderTmoveHandler;
+          $sliderHandle.bind('touchstart', sliderTdownHandler = function(tsEvt) { // factor code with aboves
+            sliding = true;
+
+            startZooming();
+            $sliderHandle.addClass("active");
+
+            var lastMove = 0;
+            windowBind('touchmove', sliderTmoveHandler = function(tmEvt) {
+              var now = tmEvt.timeStamp;
+
+              if (now > lastMove + 10) {
+                lastMove = now;
+              } else {
```

Comment:
Not crazy about this; just put 'setSliderFromMouse' inside the 'if' block so that the 'else' can be removed.